src/res/failcode.en.json

```diff
@@ -575,11 +575,13 @@
     "nmTentGt.tents": "There are too many tents around a tree.",
     "nmTentLt.tents": "There aren't enough tents around a tree.",
     "nmTentNone.tents": "A tree has no tent.",
+    "nmTouchNe.tslither": "The number is not equal to the number of separate visits to the cell's edges/vertices",
```

Comment:
All fail messages should end with a period.

Reply:
Fixed